package.json

```diff
@@ -19,7 +19,9 @@
     "start": "react-scripts start",
     "build": "react-scripts build",
     "test": "react-scripts test",
-    "eject": "react-scripts eject"
+    "eject": "react-scripts eject",
+    "lint":"eslint src/**/*.js",
+    "lint-fix":"eslint src/**/*.js --fix"
```

Comment:
you dont need this line

Reply:
you just need to do step 4 of this tutorial https://www.digitalocean.com/community/tutorials/workflow-auto-eslinting

Reply:
i add only 23 and 24 lines